bamboo-specs/specs/deploy-cumulus-orca-buckets.yaml

```diff
@@ -100,6 +100,7 @@ variables:
   DEPLOYMENT: sandbox
   AWS_DEFAULT_REGION: us-west-2
   CUMULUS_AWS_DEFAULT_REGION: us-west-2
+  CUMULUS_ACCOUNT_ID: REPLACE ME
```

Comment:
Instead of us inputting yet another variable to run this bamboo pipeline consider doing a lookup of this value similar to https://github.com/nasa/cumulus-orca/blob/67ebf253c1b519d17bd7deeb210eca808e85d6c4/integration_test/shared/setup-orca.sh#L13 in the  `integration_test/shared/setup-dr-buckets.sh` script since we have all the items needed to login to the Cumulus account.

Reply:
Tested and this works to get the cumulus account ID